public/js/cita.json

```diff
@@ -4320,8 +4320,8 @@
     "booking_location_id": "234fd904-6288-49a9-8d59-f46439ab9060"
   },
   {
-    "title": "Bromley",
-    "address": "Central Library\nHigh Street\nBROMLEY\nBR1 1EX",
+    "title": "Bromley (Central Library)",
```

Comment:
I went for parenthesis as the other location is "Bromley (Community House)" and I'm not sure "Bromley Community House" would work.